cirq/work/observable_measurement.py

```diff
@@ -31,6 +33,8 @@
     import cirq
     from cirq.value.product_state import _NamedOneQubitState
 
+MAX_REPETITIONS_PER_JOB = 3_000_000
```

Comment:
Where does this value come from? Might be useful to add a comment about this.

Reply:
documented using the `document` thing